app/assets/javascripts/app/views/aspect_membership_view.js

```diff
@@ -13,12 +13,20 @@ app.views.AspectMembership = app.views.AspectsDropdown.extend({
 
   events: {
     "click ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
-    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler"
+    "keypress ul.aspect_membership.dropdown-menu > li.aspect_selector": "_clickHandler",
+    "click ul.aspect_membership.dropdown-menu > li.newItem": "showModal"
   },
 
   initialize: function() {
     this.list_item = null;
     this.dropdown  = null;
+    if(this.$(".newAspectContainer").length > 0){
```

Comment:
No, I guess we would have to use jscs to test that. However, I agree that `if (condition) {` would be better.